website/util/metrics.py

```diff
@@ -49,6 +49,7 @@ class CampaignSourceTags(Enum):
     ErpChallenge = campaign_source_tag('erp_challenge')
     OsfRegisteredReports = campaign_source_tag('osf_registered_reports')
     Osf4m = campaign_source_tag('osf4m')
+    AGUconference2023 = campaign_source_tag('acu_conference_2023')
```

Comment:
```suggestion
    AguConference2023 = campaign_source_tag('agu_conference_2023')
```
Case consistency, typo